### ISSUE TITLE
Port attributes of abstract shell surface to attribute table

### ISSUE THREAD

**Issue by neithernut** [waysome/waysome#637]
This includes removing teh old getter/setter functions.

**Comment by matthiasbeyer** [comment]
I can do this, if you want.

**Comment by matthiasbeyer** [comment]
Will wait for #638 / #643

**Comment by matthiasbeyer** [comment]
Shall width and height members be part of the `struct ws_abstract_shell_surface`? Should this be done in another PR, maybe?

**Comment by neithernut** [comment]
I'm not yet quite sure about this, would decide this later.

**Comment by matthiasbeyer** [comment]
So I'll submit my attribute-table PR now, but note that it might should wait on a decision on this...
